src/explore-education-statistics-frontend/src/pages/_error.tsx

```diff
@@ -73,7 +73,7 @@ class ErrorPage extends Component<Props> {
       : ErrorPage.statusCodeTitles[statusCode];
 
     return (
-      <Page>
+      <Page title={pageTitle} hideTitle>
```

Comment:
Merge in master as a recent change from James has put everything in the render method and has implications here